workers/unity/Assets/Playground/Scripts/MonoBehaviours/ProcessSpinnerColorChange.cs

```diff
@@ -1,69 +1,69 @@
 using System.Collections.Generic;
-using Generated.Playground;
 using Improbable.Gdk.GameObjectRepresentation;
-using Playground;
 using UnityEngine;
-using Color = Generated.Playground.Color;
 
-public class ProcessSpinnerColorChange : MonoBehaviour
+namespace Playground.MonoBehaviours
```

Comment:
Why does this have a namespace now? But not the others?

Reply:
isn't he fixing it for a multiple fileS? for some reason some files in the playground don't have a namespace :(

Reply:
Yeah there were some missing namespaces